cpp/src/parquet/types.h

```diff
@@ -347,10 +350,14 @@ class PARQUET_EXPORT TimestampLogicalType : public LogicalType {
  public:
   static std::shared_ptr<const LogicalType> Make(bool is_adjusted_to_utc,
                                                  LogicalType::TimeUnit::unit time_unit,
+                                                 bool is_from_converted_type = false,
                                                  bool force_set_converted_type = false);
   bool is_adjusted_to_utc() const;
   LogicalType::TimeUnit::unit time_unit() const;
 
+  /// \brief If true, will not set LogicalType in Thrift metadata
```

Comment:
This isn't true anymore -- I'll remove this comment and the `is_serialized` method

Reply:
Sorry, GitHub was hiding files from me so this is indeed what is going on in this patch. Will take a closer look